lib/import/helpers/brca/providers/rq3/rq3_helper.rb

```diff
@@ -7,6 +7,22 @@ module Rq3
           module Rq3Helper
             include Import::Helpers::Brca::Providers::Rq3::Rq3Constants
 
+            def check_positive_record?
+              ['P', '?', 'UV', 'PATHOGENIC'].include? @posnegtest.upcase
```

Comment:
@shilpigoeldev - could we use %w[] here?

Reply:
Thanks @ollietulloch , done now .